posthog/async_migrations/test/test_0007_persons_and_groups_on_events_backfill.py

```diff
@@ -16,6 +16,8 @@
 from posthog.models.utils import UUIDT
 from posthog.test.base import ClickhouseTestMixin, run_clickhouse_statement_in_parallel
 
+pytestmark = pytest.mark.async_migrations
```

Comment:
Assuming this needs to be added to each async migration now?

Reply:
Yes, well, that's been the case already – I just made it simpler by only requiring it in one place for the whole file instead of for each individual test